src/Symfony/Component/Serializer/composer.json

```diff
@@ -34,7 +34,7 @@
         "symfony/http-kernel": "^4.4|^5.0|^6.0",
         "symfony/mime": "^4.4|^5.0|^6.0",
         "symfony/property-access": "^5.4|^6.0",
-        "symfony/property-info": "^5.3.13|^6.0",
+        "symfony/property-info": "^5.4.24|^6.0",
```

Comment:
"^5.4.24|^6.2.11" I guess, but can you confirm that 6.3.0 contains the fix we're needed here?

Reply:
I'm wondering why this change is needed at all: this PR patches only Serializer, not PropertyInfo.
Do you remember?

Reply:
Yes, this fix requires #49557 to work.

Reply:
But yeah, you were right, it's "^5.4.24|^6.2.11", got a little confused for a second.

Reply:
Done